developerportal/apps/events/models.py

```diff
@@ -139,7 +139,7 @@ def get_context(self, request):
     @property
     def events(self):
         """Return events in chronological order"""
-        return get_combined_events(self)
+        return get_combined_events(self, start_date__gte=datetime.date.today())
```

Comment:
Now that I think about it, maybe we should be filtering on `end_date` here? i.e. filtering out events which have finished, rather than those which have started.

Reply:
Is now a decent time to slot in some tests for these methods?

Reply:
Yeah, great points. I'll get on these in a bit!

Reply:
We’ve agreed to stick with `start_date` for now and raise a ticket to use `end_date` and possibly make it required (discussion needed?) I’m writing tests for this method now, thanks!